website/src/components/MiniBlog/styles.ts

```diff
@@ -98,7 +98,25 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
   actionWrapper: {
     justifyContent: "space-between",
-  }
+    [theme.breakpoints.down("sm")]: {
+      display: "block",
+    },
+  },
+  tabWapper: {
+    display: "flex",
+    overflow: "scroll",
+    width: "70%",
+    [theme.breakpoints.down("sm")]: {
+      display: "inlineBlock",
```

Comment:
is it `inline-block`?

Reply:
material UI supports camelCase, it won't support` -`

Reply:
That is for left-hand side CSS property not for the value